master/buildbot/util/__init__.py

```diff
@@ -252,7 +252,7 @@ def bytes2NativeString(x, encoding='utf-8'):
     @param encoding: an optional codec, default: 'utf-8'
     @return: a string of type C{str}
     """
-    if isinstance(x, bytes) and str != bytes:
+    if isinstance(x, bytes) and type("") != type(b""):
```

Comment:
I wanted to leave the future.utils constructs out of this function as much as possible,
and to future proof things, for example whenever Python 4 comes out.